adafruit_midi/control_change_values.py

```diff
@@ -46,13 +46,15 @@
 EXPRESSION = 11
 SUSTAIN_PEDAL = 64
 PORTAMENTO = 65
+PORTAMENTO_TIME = 37
```

Comment:
Should this be `5`?

Reply:
Also, would you mind moving it, whatever the value is/should be so that it's ordered by value?  Thanks!

Reply:
`5` is for the coarse adjustment value, `37` is the fine adjustment value. Will adjust & order by numeric value.

Reply:
Sounds good!  I'll mark this as resolved and wait for someone more knowledgeable make the call :D